Engine/Component/Component.h

```diff
@@ -20,7 +20,8 @@ class Component
 		UI,
 		SCRIPT,
 		TRANSFORM2D,
-		ANIMATION
+		ANIMATION,
+		AUDIO_SOURCE
```

Comment:
Talk with @marcorod94 @vidalmelero @mariofv  to decide which component is component 9 and which is component 10 (audio/collider/UI)